src/authentication/dto/authentication-response.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { User } from '@prisma/client';
+import { Exclude, Transform } from 'class-transformer';
+
+export class AuthenticationResponseDto implements User {
+  id: number;
+  email: string;
+  name: string;
+
+  @Transform(({ value: phoneNumber }) => {
```

Comment:
An anonymous function here is not the best option as it does not tell what it does. Use named functions so it is easier to understand the code.